build-tools/packages/build-tools/src/common/fluidUtils.ts

```diff
@@ -98,19 +136,32 @@ const configExplorer = cosmiconfigSync("fluidBuild", {
 
 /**
  * Loads an IFluidBuildConfig from the fluidBuild property in a package.json file, or from fluidBuild.config.[c]js.
+ * Throw if not found.
  *
  * @param rootDir - The path to the root package.json to load.
  * @param noCache - If true, the config cache will be cleared and the config will be reloaded.
  * @returns The fluidBuild section of the package.json.
  */
+export function loadFluidBuildConfig(rootDir: string, noCache = false): IFluidBuildConfig {
```

Comment:
Is the only difference between the load and get functions that load throws and get doesn't? Can they be consolidated and calling code updated to work consistently (i.e. always check undefined or handle exception)?

Reply:
I could, but didn't want to change too many code site.

Reply:
I was thinking of rewriting to whole init/infer flow to clean it up with some fluid repo specific behavior, and reduce multiple calls to `getPackages` and ways to detect package managers, but didn't want to churn the code with this PR and impact other tools that share this code.   We can talk about how (and when) we want to clean this up.